vm_scripts/networks.py

```
@@ -0,0 +1,594 @@
+            # init helper
+            content = service_instance.RetrieveContent()
+            vm_helper = VmHelper()
+
+            # get host name/ip
+            host_response = vm_helper.get_obj(content, [vim.HostSystem], host_name)
+            if host_response['status'] == "success":
+                host = host_response['res']
+            else:
+                return host_response
+
+            if method is 'create':
```

Ruff (0.8.2)

Use `==` to compare constant literals

Replace `is` with `==`

(F632)